src/app/core/services/issue.service.ts

```diff
@@ -76,7 +76,7 @@ export class IssueService {
   getLatestIssue(id: number): Observable<Issue> {
     return this.githubService.fetchIssueGraphql(id).pipe(
       map((response: GithubIssue) => {
-        this.createAndSaveIssueModel(response);
+        this.createAndSaveIssueModels([response]);
         return this.issues[id];
       }),
       catchError((err) => {
```

Comment:
The methods `getLatestIssue` and `getIssue` are not in use throughout the application. Let's take a look at unused methods in a separate issue and remove them from the codebase in phases.

Reply:
Yep, there can be a separate PR that contributes to issue #72